todo_app/flask_config.py

```diff
@@ -7,3 +7,5 @@ def __init__(self):
         self.SECRET_KEY = os.environ.get('SECRET_KEY')
         if not self.SECRET_KEY:
             raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+        self.LOG_LEVEL = os.environ.get('LOG_LEVEL')
```

Comment:
One thing we could do here (and/or in your Terraform) is to set a default level in case someone doesn't set one, for example with:
```suggestion
        self.LOG_LEVEL = os.environ.get('LOG_LEVEL', 'INFO')
```

Reply:
Thanks Jack